posnev.c

```diff
@@ -0,0 +1,26 @@
+//Anya Lauria
```

Comment:
Nice job! You implemented the assignment correctly - my only suggestion is to reconsider how you handle 0 next time!